libvirt/tests/src/libvirt_mem.py

```
@@ -540,12 +660,34 @@ def modify_domain_xml():
             old_mem_total = vm.get_totalmem_sys(online)
             logging.debug("Memtotal on guest: %s", old_mem_total)
             session.close()
+
+        # Capture baseline for verification
+        if attach_option or detach_option:
```

Ruff (0.15.15)

[error] Undefined name `detach_option`

(F821)